spec/cucumber_factory/steps_spec.rb

```diff
@@ -428,6 +428,13 @@
       obj.attributes[:total].to_s.should == "45.6"
     end
 
+    it "should allow to set file attributes with the FILE syntax" do
+      invoke_cucumber_step('there is a plain Ruby class with the file <FILE:spec/assets/file.txt>')
+      obj = PlainRubyClass.last
+      obj.attributes[:file].should be_a(File)
+      obj.attributes[:file].read.should == "This is a test file.\n"
+    end
+
```

Comment:
Please add some other tests:
* for FactoryBot
* for Machinist
* for ActiveRecord
* with the single quote Syntax
* with the double quote Syntax
* With the table syntax
* With the doc string syntax
* combined with the "update" step

Lastly, it would be great if we could test a real CarrierWave attribute. But if that is too much hassle, tell me and I'll test it manually in a real world project.

Reply:
One more thing: Please test the use of symlinked paths.
For example, we often have a folder like `spec/fixtures/` containing files like `sample1.txt` and `sample2.txt`. To reduce the repository's overall size, `sample2.txt` might be symlinked to `sample1.txt`